RAGchain/benchmark/dataset/mr_tydi.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, run_pipeline: BaseRunPipeline,
                            ['MRR'])
         languages = ['arabic', 'bengali', 'combined', 'english', 'finnish',
                      'indonesian', 'japanese', 'korean', 'russian', 'swahili', 'telugu', 'thai']
+        language = text_modifier(language)[0].lower()
```

Comment:
use first index of modified texts?? Then what is the point to use text_modifier function? 
text_modifier function is made for modifying text to lots of variations, and return to string text. 
In that process, you can get lower case word for comparing between lower case word and `languages` elements.

`text_modifier(language)[0].lower()` is literally same as `language.lower()`